.changeset/moody-frogs-send.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/container-utils": minor
+---
+
+Deprecate DeltaManagerProxyBase
+
+Deprecates DeltaManagerProxyBase. No replacement API is intended.
```

Comment:
Thoughts? Along similar lines to other comments I've made recently about clearer release notes both for us and for external consumers.

```suggestion
DeltaManagerProxyBase is only used internally in FluidFramework code and will no longer be exported in a future release. No replacement API is intended for external consumers.
```